### PR TITLE
About: HTML-escape third party license text.

### PR DESCRIPTION
**Pull request by mkrautz**
Fixes mumble-voip/mumble#2448

### CODE REVIEW

**Comment by mkrautz** [comment]
PTAL @hacst @Kissaki @fwaggle @Natenom

**Comment by Kissaki** [comment]
`Qt::escape` [is obsolete](http://doc.qt.io/qt-5/qt-obsolete.html#escape).

Use [`QString::toHtmlEscaped()`](http://doc.qt.io/qt-5/qstring.html#toHtmlEscaped) instead.

File: src/mumble/About.cpp [hunk 1]
```diff
@@ -28,9 +28,9 @@ AboutDialog::AboutDialog(QWidget *p) : QDialog(p) {
 	for(int i = 0; !licenses3rdParties[i].isEmpty(); ++i)
 	{
 		qtb3rdPartyLicense->append(QString::fromLatin1("<h3>%1 (<a href=\"%2\">%2</a>)</h3><pre>%3</pre>")
-				.arg(QString::fromLatin1(licenses3rdParties[i].name))
-				.arg(QString::fromLatin1(licenses3rdParties[i].url))
-				.arg(QString::fromLatin1(licenses3rdParties[i].license)));
+				.arg(Qt::escape(QString::fromLatin1(licenses3rdParties[i].name)))
```
Comment (Kissaki) [inline src/mumble/About.cpp]: `Qt::escape` [is obsolete](http://doc.qt.io/qt-5/qt-obsolete.html#escape).

Use [`QString::toHtmlEscaped()`](http://doc.qt.io/qt-5/qstring.html#toHtmlEscaped) instead.

Comment (mkrautz) [inline src/mumble/About.cpp]: Not available in Qt 4.

**Comment by Kissaki** [comment]
LGTM

**Pull request merged**